cargo-quickinstall/src/main.rs

```diff
@@ -285,14 +286,21 @@ fn install_crate(crate_name: &str, version: &str, target: &str) -> Result<(), In
 }
 
 fn main() -> Result<(), Box<dyn std::error::Error + Send + Sync + 'static>> {
-    let args = std::env::args().collect::<Vec<_>>();
-    let crate_name = if let Some(true) = args.get(1).map(|a| a == "quickinstall") {
-        args.get(2)
-    } else {
-        args.get(1)
-    };
-
-    let crate_name = crate_name.ok_or("USAGE: cargo quickinstall CRATE_NAME")?;
+
+    let mut args = Arguments::from_env();
+    if args.contains("--version") {
+        let version_self = env!("CARGO_PKG_VERSION");
+        println!("{}", version_self);
```

Comment:
Ah. I was thinking of adding a --version flag that behaves like how `cargo install --version` does

Reply:
I wonder if we could make a ["--print-version", "-V"] flag that does this.

Reply:
I suspected that :)